bigtable/tests/system.py

```diff
@@ -295,6 +295,76 @@ def test_delete_column_family(self):
         # Make sure we have successfully deleted it.
         self.assertEqual(temp_table.list_column_families(), {})
 
+    def test_retry(self):
+        import subprocess, os, stat, platform
+        from google.cloud.bigtable.client import Client
+        from google.cloud.bigtable.instance import Instance
+        from google.cloud.bigtable.table import Table
+
+        # import for urlopen based on version
+        try:
+            # python 3
+            from urllib.request import urlopen
+        except ImportError:
+            # python 2
+            from urllib2 import urlopen
+
+
+        TEST_SCRIPT = 'tests/retry_test_script.txt'
+        SERVER_NAME = 'retry_server'
+        SERVER_ZIP = SERVER_NAME + ".tar.gz"
+
+        def process_scan(table, range, ids):
+            range_chunks = range.split(",")
+            range_open = range_chunks[0].lstrip("[")
+            range_close = range_chunks[1].rstrip(")")
+            rows = table.read_rows(range_open, range_close)
+            rows.consume_all()
+
+        # Download server
+        MOCK_SERVER_URLS = {
+            'Linux': 'https://storage.googleapis.com/cloud-bigtable-test/retries/retry_server_linux.tar.gz',
+            'Darwin': 'https://storage.googleapis.com/cloud-bigtable-test/retries/retry_server_mac.tar.gz',
+        }
+
+        test_platform = platform.system()
+        if (test_platform not in MOCK_SERVER_URLS):
+            self.fail("Retry server not available for platform " + test_platform)
```

Comment:
This will make the tests flatly fail on Windows (and therefore AppVeyor), or any other unrecognized platform. This seems harsh. Can we just _skip_ the test instead?

```python
if test_platform not in MOCK_SERVER_URLS:
    self.skip('Retry server not available for platform {0}.'.format(test_platform))
```